packages/client-core/src/components/Settings/DeleteAccountScreen.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and
+provide for limited attribution for the Original Developer. In addition,
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2025
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { CheckSm } from '@ir-engine/ui/src/icons'
+import { motion } from 'motion/react'
+import React, { useState } from 'react'
+
+interface DeleteAccountScreenProps {
+  navigateTo: (screen: string) => void
+  onClose?: () => void
+}
+
+const DeleteAccountScreen: React.FC<DeleteAccountScreenProps> = ({ navigateTo, onClose }) => {
+  const [showConfirmation, setShowConfirmation] = useState(true)
+  const [showSuccess, setShowSuccess] = useState(false)
+
+  const handleDelete = () => {
+    setShowConfirmation(false)
+    setShowSuccess(true)
+  }
+
+  const handleStayHere = () => {
+    navigateTo('account')
+  }
+
+  const handleClose = () => {
+    onClose?.()
+  }
+
+  if (showSuccess) {
+    return (
+      <div className="flex h-full flex-col items-center justify-center space-y-8 text-center">
+        {/* Success Checkmark */}
+        <motion.div
+          initial={{ scale: 0 }}
+          animate={{ scale: 1 }}
+          transition={{ duration: 0.3, delay: 0.2 }}
+          className="flex h-16 w-16 items-center justify-center rounded-full"
+          style={{
+            background: 'linear-gradient(145deg, rgba(255,255,255,0.2), rgba(255,255,255,0.1))',
+            border: '1px solid rgba(255, 255, 255, 0.15)'
+          }}
+        >
+          <CheckSm className="h-8 w-8 text-white" />
+        </motion.div>
+
+        {/* Success Message */}
+        <motion.div
+          initial={{ opacity: 0, y: 20 }}
+          animate={{ opacity: 1, y: 0 }}
+          transition={{ duration: 0.3, delay: 0.4 }}
+          className="space-y-2"
```

Comment:
Can you use `gap` instead of `space`. `grid gap-y-2`

https://v3.tailwindcss.com/docs/space#limitations

[Gap vs Space](https://www.reddit.com/r/tailwindcss/comments/zrd1nc/comment/j13dh0l)